lib/modules/manager/ansible-galaxy/extract.ts

```diff
@@ -13,7 +13,7 @@ export function getSliceEndNumber(
   if (start < 0 || start > numberOfLines - 1) {
     return -1;
   }
-  let nearestEnd = numberOfLines - 1;
+  let nearestEnd = numberOfLines;
```

Comment:
I am unsure of this change. Not because I'm unsure it's right, but because I'm not sure why it was -1 in the first place too

Reply:
I think the `getSliceEndNumber` function is to get the end line number to slice for each `colection` or `roles` content.

If the `slice` function is used with a range specified, such as `slice(start,end)`, `end` is not included. [doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice)

I think that a misunderstanding of the specification of the `slice` function caused it to set `numberOfLines - 1`.